xds/internal/client/rds_test.go

```diff
@@ -500,3 +500,125 @@ func (s) TestHostFromTarget(t *testing.T) {
 		})
 	}
 }
+
+func TestMatchTypeForDomain(t *testing.T) {
+	tests := []struct {
+		d    string
+		want domainMatchType
+	}{
+		{
```

Comment:
Maybe make these entries fit in single lines like the ones in `TestMatch`, just for consistency?

Reply:
Done